colossus/src/main/scala/colossus/service/ServiceClient.scala

```diff
@@ -273,6 +275,8 @@ extends Controller[O,I](codec, ControllerConfig(config.pendingBufferSize, config
       //don't allow any new requests, appear as if we're dead
       s.handler(Failure(new NotConnectedException("Not Connected")))
     } else if (isConnected || !failFast) {
+      requestBufferSize.set(queueSize)
+      requestBufferRatio.add(((queueSize.toDouble / config.pendingBufferSize.toDouble) * 100).toInt, Map("full" -> outputQueueFull.toString))
```

Comment:
These two metrics seem to be basically tracking the same thing in slightly different ways, so I think we should cut one of them out, probably the size gauge.

Also, the "full" tag doesn't seem very useful, and insead the metric should be adding the `hpTags` to add host/port as tags.  Otherwise the values from every client will get merged together.